cl/search/documents.py

```diff
@@ -442,16 +441,26 @@ def prepare_political_affiliation(self, instance):
             pa.get_political_party_display()
             for pa in instance.political_affiliations.all()
             if pa
-        ]
+        ] or None
 
     def prepare_political_affiliation_id(self, instance):
         return [
             pa.political_party
             for pa in instance.political_affiliations.all()
             if pa
-        ]
+        ] or None
 
     def prepare_aba_rating(self, instance):
         return [
             r.get_rating_display() for r in instance.aba_ratings.all() if r
-        ]
+        ] or None
+
+    def prepare_text(self, instance):
+        text_template = loader.get_template("indexes/person_text.txt")
+        return text_template.render({"item": instance}).translate(null_map)
```

Comment:
I have my same question here about whether we can get rid of this and still have a useful search engine. Seems like maybe?

Reply:
Yeah, we can remove this after adding a few fields to the `PersonDocument` class.